mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
+/* Math.random function returns a pseudo random value between 0 to 1 and multiplies it by 10.*/
 function getRandomNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+/* Concat method is used to merge two or more arrays and it does not change the existing array but returns a new array.*/
```

Comment:
It can be used as a method on an array, but here it is being used on a string. It behaves in the same way on both types.

Reply:
Changed the comment to mention strings as well as arrays. :)